api/models/process.go

```diff
@@ -67,6 +68,54 @@ func GetServices(app string) ([]*ecs.Service, error) {
 	return dres.Services, nil
 }
 
```

Comment:
include this note:

//NOTE: the DescribeService Service.Event contains a maximum of 100 of the latest events
// http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Service.html